menpo3d/vtkutils.py

```diff
@@ -0,0 +1,130 @@
+import numpy as np
+import vtk
+from vtk.util.numpy_support import (numpy_to_vtk, numpy_to_vtkIdTypeArray,
+                                    vtk_to_numpy)
+from menpo.shape import TriMesh
+
+
+def trimesh_to_vtk(trimesh):
+    r"""Return a `vtkPolyData` representation of a :map:`TriMesh` instance
+
+    Parameters
+    ----------
+    trimesh : :map:`TriMesh`
+        The menpo :map:`TriMesh` object that needs to be converted to a
+        `vtkPolyData`
+
+    Returns
+    -------
+    `vtk_mesh` : `vtkPolyData`
+        A VTK mesh representation of the Menpo :map:`TriMesh` data
+
+    Raises
+    ------
+    ValueError:
+        If the input trimesh is not 3D.
+    """
+    if trimesh.n_dims != 3:
+        raise ValueError('trimesh_to_vtk() only works on 3D TriMesh instances')
+
+    mesh = vtk.vtkPolyData()
+    points = vtk.vtkPoints()
+    points.SetData(numpy_to_vtk(trimesh.points, deep=1))
+    mesh.SetPoints(points)
+
+    cells = vtk.vtkCellArray()
+    cells.SetCells(trimesh.n_tris,
+                   numpy_to_vtkIdTypeArray(
+                       np.hstack((np.ones(trimesh.n_tris)[:, None] * 3,
+                                  trimesh.trilist)).astype(np.int64).ravel(),
```

Comment:
@patricksnape I'll change to int32 and let all CI's run, haven't got a Windows box to hand to test with